http-server-netty/src/test/groovy/io/micronaut/http/server/netty/configuration/NettyHttpServerConfigurationSpec.groovy

```diff
@@ -138,4 +197,24 @@ class NettyHttpServerConfigurationSpec extends Specification {
         cleanup:
         beanContext.close()
     }
+
+    static class BasicConditionContext implements ConditionContext {
```

Comment:
What purpose is this class serving?

Reply:
It was used to test `Condition`. No longer needed, I will removed it.